docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -96,14 +96,51 @@ releases of Ansible, there can sometimes be exceptions for critical issues.
 
 .. _GitHub: https://github.com/ansible/ansible
 
+.. _changelogs_how_to:
+
 Changelogs
 ~~~~~~~~~~~~~~~~~~
 
-Since 2.5, we've logged changes to ``stable-<version>`` git branches at ``stable-<version>/changelogs/CHANGELOG-v<version>.rst``.
-For example, here's the changelog for 2.5_ on GitHub.
+Since 2.5, we have switched to generating changelogs based on fragments. When creating new features or fixing bugs, create a changelog fragment describing the change. A changelog entry is not needed for new modules or plugins. Details for those items will be generated from the module documentation.
```

Comment:
I like this content a lot, but I'm not sure it belongs in the reference section under Releases and Maintenance, which is more targeted at users who are wondering when 2.1 was released or when the can expect the next version to come out. 

I'll think about where this content would fit best - probably somewhere in the Developer Guide. 

If I can't come up with a better location, and we add this content in this location, I'd change the beginning of this sentence, so it goes with the restored sentences above: "We made this change so we could generate changelogs based on fragments." The rest of the content looks great.

Reply:
Originally I found this here and just expanded it. A better home is probably in `development_process.rst` and link to it there from `release_and_maintenance.rst`, the opposite of how it is now.